c/README.md

```diff
@@ -0,0 +1,4 @@
+# How to run
+
+- Make sure you have a compiler installed on your system.
```

Comment:
Would you be so kind to add a small guide (or link to a small guide) on how this is done? Looks great otherwise.

Reply:
No problemo, updated. But i was wondering if this project targets newbies wouldn't it also be nice to include some sort of hyperlink to a DSA course? I imagine it would be hard doing AOC without basic DSA knowledge, or does it rather target people moving onto other language who already do know basic computer programming?

Reply:
You're right. It may be nice to have a little list of links to learning resources on commonly used data structures and algorithms. Please feel free to open up an issue for this!